lib/features/upload/data/model/upload_file_arguments.dart

```diff
@@ -8,13 +9,15 @@ import 'package:tmail_ui_user/features/upload/domain/model/upload_task_id.dart';
 class UploadFileArguments with EquatableMixin {
 
   final DioClient dioClient;
+  final FileUtils fileUtils;
```

Comment:
any concerned about isolate?

Reply:
Tested it is not affected, functions in FileUtils access native, storage so not affected by isolate like network

Reply:
how about failed case, toast also be in weird or not?

Reply:
It has nothing to do with this.